packages/lab/src/stepped-tracker/TrackerStep/TrackerStep.tsx

```diff
@@ -35,14 +37,30 @@ export interface TrackerStepProps extends ComponentPropsWithoutRef<"li"> {
    * If the stage is completed or active, the status prop will be ignored
    */
   status?: StatusOptions;
+  /**
+   * The nesting depth of the TrackerStep
+   */
+  depth?: Depth;
```

Comment:
Not sure whether this is compared with `NavigationItem`'s `level`.

Worth to point out this will take no effect when horizontal?